packages/datadog_session_replay/example/lib/screens/touch_privacy_screen.dart

```diff
@@ -0,0 +1,88 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025-Present Datadog, Inc.
+
+import 'package:datadog_session_replay/datadog_session_replay.dart';
+import 'package:flutter/material.dart';
+
+class TouchPrivacyScreen extends StatefulWidget {
+  const TouchPrivacyScreen({super.key});
+
+  @override
+  State<TouchPrivacyScreen> createState() => _ImagesScreenState();
+}
+
+class _ImagesScreenState extends State<TouchPrivacyScreen> {
```

Comment:
Should it be `_TouchPrivacyScreenState`?
```suggestion
class _TouchPrivacyScreenState extends State<TouchPrivacyScreen> {
```